App/src/pages/dossier/Dossiers.tsx

```diff
@@ -263,7 +266,25 @@ export default function Dossiers() {
                         setCurrentPage={setCurrentPage}
                         currentPage={currentPage}
                         totalResults={totalResults}
+                        useIcons={false}
                     />
+
+                    <span style={{ alignSelf: "flex-end", width: "fit-content", height: "fit-content" }}>
+                        <Button
```

Comment:
Can you put this button underneath or besides the "Back" button at the top of the page so the navigation buttons can be grouped together?